examples/README.md

```diff
@@ -0,0 +1,9 @@
+# BluePyEModel Examples
+
+This directory includes various examples demonstrating how to use BluePyEModel:
+
+* L5PC:  Provides a guide for setting up and running the E-Model building pipeline locally or on a Slurm cluster.
```

Comment:
Could add this line: 
"This is an example based on the juvenile (P14) rat somatosensory cortex layer 5 thick-tufted pyramidal cell (L5TPC) e-model optimisation of the  [SSCxEModelExamples](https://github.com/BlueBrain/SSCxEModelExamples/tree/main/optimization) repository"

Reply:
> pipeline locally or on a Slurm cluster.

"pipeline locally on a personal computer or on a Slurm cluster."

Reply:
done in one of the last commits